packages/web-components/src/components/rux-global-status-bar/rux-global-status-bar.scss

```diff
@@ -7,7 +7,8 @@
     top: 0;
     left: 0;
     height: 100px;
-    width: 100%;
+    width: -webkit-fill-available;
```

Comment:
@jonathantneal any cons to using this even though its experimental? it looks like `fill-available` was renamed to `stretch` but nobody support it yet?

Reply:
@markacianfrani, it should be `stretch`, and only with the prefixed values as fallbacks if [Autoprefixer](https://autoprefixer.github.io/) is not also being applied.

```css
:host {
    width: -moz-available;
    width: -webkit-fill-available;
    width: stretch;
}
```

Reply:
Added!